app/views/teams/_show_mobile.html.erb

```diff
@@ -54,12 +53,12 @@
         <% end %>
       <% end %>
     </div>
-    <div id="user" data-teams-tab-change-target="teamContent" class="" data-tab="user">
+    <div id="user" data-teams-tab-change-target="teamContent" class="pt-6" data-tab="user" data-controller="user-show">
       <div class="my-4 flex hidden justify-start" data-target="user-show.allUsersSection">
         <span class="text-xs font-semibold text-primary">All users</span>
       </div>
       <div class="my-4 flex justify-end hidden" data-user-show-target="showAllUsers">
-        <a href="#" data-action="click->user-show#toggleAllUsers" class="hidden flex items-center justify-center text-base font-normal">
+        <a href="#" data-action="click->user-show#toggleAllUsers" class="flex items-center justify-center text-base font-normal">
```

Comment:
This is intentionally kept as hidden because the feature is not implemented yet

Reply:
user-show controller is removed, can be used later if needed